File: src/commands/proxy/config/langchain.js
import { ChatOpenAI } from "@langchain/openai";

const ASIMOV = new ChatOpenAI({
  configuration: {
    baseURL: "http://127.0.0.1:1920/v1"
  },
  apiKey: "langchain",
  model: "openrouter/free",
});

//console.log(await ASIMOV.invoke("Tell me about Isaac Asimov"));

File: src/commands/proxy/config/langchain.py
from langchain_openai import ChatOpenAI

ASIMOV = ChatOpenAI(
    base_url="http://127.0.0.1:1920/v1",
    api_key="langchain",
    model="openrouter/free",
)

#print(ASIMOV.invoke("Tell me about Isaac Asimov"))
